rails/bin/dev

```diff
@@ -1,8 +1,3 @@
 #!/usr/bin/env sh
 
-if ! gem list foreman -i --silent; then
-  echo "Installing foreman..."
-  gem install foreman
-fi
-
-exec foreman start -f Procfile.dev "$@"
```

Comment:
This setup was done in this manner for a reason. See the `foreman` gem notes above.